README.md

```diff
@@ -58,12 +67,12 @@ target_link_libraries( ${PROJECT_NAME} PUBLIC
 ### Importing as Python Module
 > [!IMPORTANT]\
 >Python module support is currently only available for x86/amd devices
-To import this library as a python module the following is necessary
+To import this library as a python module the following is necessary.
 
 ```python
 # This path can be added via the sys module or by directly appending the PYTHON_PATH environment variable.
 import sys
-sys.path.append('/opt/carma/lib/')
+sys.path.append('/path/to/library')
```

Comment:
Which python library? What is the library path?

Reply:
Path to the library you build using the new instructions provided.

Reply:
Is this path to python library or carma-clock library?